### PR TITLE
Updated all images in KM to 2x

### PR DESCRIPTION
**Pull request by dmcalester**
All images have been updated to 2x
Fixes ASTRO-1107 https://rocketcom.atlassian.net/browse/ASTRO-1107

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	"scripts": {
 		"test": "echo \"Error: no test specified\" && exit 1",
 		"start": "run-p -l start:*",
+		"start:clean": "rimraf _site",
```
Comment (amalyah) [inline package.json:13]: @dmcalester What does this change do?

**Review by amalyah — COMMENTED** [review]
- Broken image links on [Status System](https://deploy-preview-134--astrouxds.netlify.app/patterns/status-system/), [Navigation](https://deploy-preview-134--astrouxds.netlify.app/patterns/navigation/), [Theme](https://deploy-preview-134--astrouxds.netlify.app/design-guidelines/theme/), [Sample App](https://deploy-preview-134--astrouxds.netlify.app/patterns/sample-app/), and [Dialog Box](https://deploy-preview-134--astrouxds.netlify.app/components/dialog-box/) pages.

- Oversized images on [Compliance](https://deploy-preview-134--astrouxds.netlify.app/design-guidelines/compliance/) (4.6.6, 4.7.3, 4.10.2, 4.12.1, 4.12.2, 4.13.6, and 4.14.2) and [Accordion](https://deploy-preview-134--astrouxds.netlify.app/components/accordion/).

- Blurry images (not sure if we can do anything about them, but thought I'd note them) on: [Visual Design](https://deploy-preview-134--astrouxds.netlify.app/design-process/visual-design/), most of the [TT&C](https://deploy-preview-134--astrouxds.netlify.app/ttc-service-ux-design/about-the-ttc-designs/) pages, most of the [GRM](https://deploy-preview-134--astrouxds.netlify.app/grm-service-ux-design/about-the-grm-designs/) pages, the[ Service Specific](https://deploy-preview-134--astrouxds.netlify.app/service-specific-ux-design/) page, [Global Status Bar](https://deploy-preview-134--astrouxds.netlify.app/components/global-status-bar/), [Buttons](https://deploy-preview-134--astrouxds.netlify.app/components/button/), [Component Sandbox](https://deploy-preview-134--astrouxds.netlify.app/components/component-sandbox/), [UI Design](https://deploy-preview-134--astrouxds.netlify.app/design-process/ui-design/), and some blurry images on the [FD Service](https://deploy-preview-134--astrouxds.netlify.app/fd-service-ux-design/about-the-fd-designs/) page.

File: package.json (hunk 1 above)
Comment (dmcalester) [reply package.json:13]: The Astro site gets built from templates (for structure/layout), markdown (for content) and static assets like images and compiled in to tradition HTML/CSS/JS structure in a `_site` directory. When testing locally the static assets can sometimes become stale, the content and code will update but an image might be out of date. `rimraf` deletes the `_site` directory so it gets rebuilt from scratch every time the local dev server starts.

**Comment by dmcalester** [comment]
> * Broken image links on [Status System](https://deploy-preview-134--astrouxds.netlify.app/patterns/status-system/), [Navigation](https://deploy-preview-134--astrouxds.netlify.app/patterns/navigation/), [Theme](https://deploy-preview-134--astrouxds.netlify.app/design-guidelines/theme/), [Sample App](https://deploy-preview-134--astrouxds.netlify.app/patterns/sample-app/), and [Dialog Box](https://deploy-preview-134--astrouxds.netlify.app/components/dialog-box/) pages.

These should be fixed

> * Oversized images on [Compliance](https://deploy-preview-134--astrouxds.netlify.app/design-guidelines/compliance/) (4.6.6, 4.7.3, 4.10.2, 4.12.1, 4.12.2, 4.13.6, and 4.14.2) and [Accordion](https://deploy-preview-134--astrouxds.netlify.app/components/accordion/).

There is a temporary solution in place, it is not ideal, but not grievous enough to hold back launch

> * Blurry images (not sure if we can do anything about them, but thought I'd note them) on: [Visual Design](https://deploy-preview-134--astrouxds.netlify.app/design-process/visual-design/), most of the [TT&C](https://deploy-preview-134--astrouxds.netlify.app/ttc-service-ux-design/about-the-ttc-designs/) pages, most of the [GRM](https://deploy-preview-134--astrouxds.netlify.app/grm-service-ux-design/about-the-grm-designs/) pages, the[ Service Specific](https://deploy-preview-134--astrouxds.netlify.app/service-specific-ux-design/) page, [Global Status Bar](https://deploy-preview-134--astrouxds.netlify.app/components/global-status-bar/), [Buttons](https://deploy-preview-134--astrouxds.netlify.app/components/button/), [Component Sandbox](https://deploy-preview-134--astrouxds.netlify.app/components/component-sandbox/), [UI Design](https://deploy-preview-134--astrouxds.netlify.app/design-process/ui-design/), and some blurry images on the [FD Service](https://deploy-preview-134--astrouxds.netlify.app/fd-service-ux-design/about-the-fd-designs/) page.

Creating a ticket for addressing blurry images. https://rocketcom.atlassian.net/browse/ASTRO-1148

**Review by rocket-github — APPROVED** [review]

**Review by cskerbo — APPROVED** [review]

**Review by amalyah — CHANGES_REQUESTED** [review]
- Still seeing accordion icons as larger than they are on the live site.
- For compliance: the max 50% width rule works well for some images, but not others. Would it be worth putting in exceptions for 4.5.3 and 4.6.3? Those images are now too small to give any value to the page in my opinion. Not worth delaying a deploy, but something to think about. May be better at that point just to remove the image from that page altogether.

**Pull request merged**